internal/pkg/model/schema.go

```diff
@@ -333,6 +322,23 @@ type PolicyLeader struct {
 	Timestamp string `json:"@timestamp,omitempty"`
 }
 
+// PolicyOutput holds the needed data to manage the output API keys
+type PolicyOutput struct {
+	ESDocument
+
+	// API key the Elastic Agent uses to authenticate with elasticsearch
+	APIKey string `json:"api_key"`
+
+	// ID of the API key the Elastic Agent uses to authenticate with elasticsearch
+	APIKeyID string `json:"api_key_id"`
+
+	// The policy output permissions hash
+	PolicyPermissionsHash string `json:"policy_permissions_hash"`
```

Comment:
I think the word `Policy` here is redundant and makes the field content larger because name is rather verbose. How about just `permissions_hash`?